### ISSUE TITLE
Add rendering for amenity=coworking_space

### ISSUE THREAD

**Issue by vdeparday** [gravitystorm/openstreetmap-carto#1628]
Coworking space and innovation hubs are/can be a big part of growing the OSM community, however they do not get rendered on the map, so it would be nice to add these at these two tags to be rendered:
http://wiki.openstreetmap.org/wiki/Tag:amenity=coworking%20space?uselang=en-US
http://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcommunity_centre

There may be some other ones used for similar spaces.

**Comment by kocio-pl** [comment]
I guess we will have a bigger coloring change, including your examples, because [we tend to agree now](https://github.com/gravitystorm/openstreetmap-carto/pull/1450#issuecomment-117227190), that filling color is not universal enough and we don't need it everywhere (too broad) or have many filling colors (that would make a map look like a rainbow), so instead we may end up with color schemes for boundaries.

**Comment by pnorman** [comment]
edited to reflect just `amenity=coworking_space` as the two tags are quite unrelated.

**Comment by pnorman** [comment]
225 uses is insufficient to add as a POI

**Comment by vdeparday** [comment]
Thanks Paul for the quick answer. Is that for technical reasons and maintenance? Because in terms of importance or meaning, I am not sure why if a tag is not used that much, it would be mean that it is not relevant to be shown? Or would that mean there is a flaw in the tag itself?

**Comment by kocio-pl** [comment]
AFAIK there are currently no policy on the amount of tag uses yet. It's just a matter of individual taste and feeling. In general the more occurrences, the more contributors agree it's worth showing on the map, of course. See the comments [in copy shop pull request](https://github.com/gravitystorm/openstreetmap-carto/pull/1615): @pnorman is more demanding in this regard, @math1985 has lower expectations, and mine are even less strict than them both.

The bottom line is you can not agree and try to prove your reasons are justified, but until we have no such policy, the people able to merge the pull requests have final say in practice.

**Comment by dieterdreist** [comment]
2015-07-03 10:48 GMT+02:00 Vivien Deparday <notifications@github.com>:

> Thanks Paul for the quick answer. Is that for technical reasons and
> maintenance? Because in terms of importance or meaning, I am not sure why
> if a tag is not used that much, it would be mean that it is not relevant to
> be shown? Or would that mean there is a flaw in the tag itself?



Completely agree, as has been pointed out previously, pure usage numbers
should not dictate which features get rendered (and haven't been in the
past, think of capital=yes ;-) ). There are fewer coworking spaces in the
world then there are trash bins. Man_made=obelisk, historic=city_gate or
historic=aqueduct (to name some pet tags of mine) will hardly take the 5000
uses hurdle, but they are very important objects in a general map because
they are typically prominent for (/determining the identity of) the place
where they stand.

**Comment by polarbearing** [comment]
amenity is a questionable namespace anyway for this feature, as it is a particular kind of office infrastructure.

**Comment by RalphCorderoy** [comment]
https://taginfo.openstreetmap.org/tags/?key=amenity&value=coworking_space#overview now says there's 547, more than double the 225 back in 2015 when this was considered. And it's quite handy when looking at a town that you know has lots of tech. companies to spot cow orkers with meeting rooms for hire instead of the normal inappropriate coffee shop. Can this request for rendering amenity=coworking_space please be reconsidered. Thanks.

**Comment by polarbearing** [comment]
There was some discussion recently in the wiki and on the tagging list to use the tag office=coworking, since most of those facilities are offices. As iD has added that as a preset, there are already 100+ uses of that version: https://taginfo.openstreetmap.org/tags/office=coworking

**Comment by RalphCorderoy** [comment]
Thanks for the information, though I don't think office=coworking is rendered either?

**Comment by borysiasty** [comment]
Regardless of the decision what tag should be used, I'd like to support the idea of rendering coworking spaces. It's now 1160 occurrences of the three versions (amenity=coworking_space + office=coworking + office=coworking_space), and I can add a next bunch od them, but I would be more motivated if they are visible on the render.

**Comment by kocio-pl** [comment]
Apart from tagging and usage, we also don't know how to render it, so this issue is troublesome.

**Comment by borysiasty** [comment]
It's a problem, indeed. I'm not good in graphics, but will think about a concept.

**Comment by RalphCorderoy** [comment]
Head and shoulders of two people facing one another, with a laptop each in between?

**Comment by kocio-pl** [comment]
So next thing would be actual icon design (apart from other problems).

**Comment by SomeoneElseOSM** [comment]
> Head and shoulders of two people facing one another, with a laptop each in between?

Will it be possible to render the beards in such a small icon though? :)

More seriously, I'm not convinced that any icon would necessarily work here.  Co-working spaces cover all sorts of enterprises (social and commercial), so I'd suggest just a "black office dot" or similar.

**Comment by Tomasz-W** [comment]
Is anybody interested in writing code with office-blue dot for this feature?

**Comment by Tomasz-W** [comment]
@kocio-pl As we have office-blue dots for all offices, and amenity=coworking_space is kind of an office, I think this issue should be re-opened and we should try with office-blue generic dot for it.

For those who like to see the numbers:
* office=coworking (coworking spaces which are office) -> 534 uses and it's rendered
* amenity=coworking_space -> 822 uses and it's not rendered

**Comment by polarbearing** [comment]
That means when we had in [February2017 100+ office=coworking](#issuecomment-281982353) and have now 534, this tag is now exploding. Amenity is still growing but much less than office. 

I would not want to support fragmenting the same feature with rendering two different tags.

**Comment by Tomasz-W** [comment]
> I would not want to support fragmenting the same feature with rendering two different tags.

They are not the same. See https://wiki.openstreetmap.org/wiki/Tag:amenity=coworking_space#Alternative_Tagging
It means that not every amenity=coworking_space is a typical office.

**Comment by polarbearing** [comment]
Current usage stats:
office=coworking (rendered as blue office dot): 1590
amenity=coworking_space: 1429
